scripts/confix/testdata/v36-config.toml

```diff
@@ -281,7 +281,7 @@ recv-rate = 5120000
 #######################################################
 [mempool]
 
-recheck = true
+# Set true to broadcast transactions in the mempool to other nodes
```

Comment:
Did you hand-edit this file, or was it generated by the node?

Reply:
I hand edited this file. Sorry I was just grepping, I probably should have checked if there was a tool for generating it

Reply:
It's fine. There isn't—I just build the node with the change, and run it to generate a new config file. I'll do that and update the PR.

Reply:
(ordinarily we shouldn't need to update these, the only reason we have to now is that the "v36" config is still not settled)